x-pack/plugins/synthetics/public/apps/synthetics/components/getting_started/simple_monitor_form.tsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFieldText,
+  EuiFormRow,
+  EuiForm,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+} from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { useFetcher } from '@kbn/observability-plugin/public';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { ServiceLocationsField } from './form_fields/service_locations';
+import { kibanaService } from '../../../../utils/kibana_service';
+import { DEFAULT_FIELDS } from '../../../../../common/constants/monitor_defaults';
+import { createMonitorAPI } from '../../state/monitor_management/api';
+import {
+  ConfigKey,
+  DataStream,
+  ServiceLocations,
+  SyntheticsMonitorWithId,
+} from '../../../../../common/runtime_types';
+import { useFormWrapped } from '../../../../hooks/use_form_wrapped';
+
+export interface SimpleFormData {
+  urls: string;
+  locations: ServiceLocations;
+}
+
+export const SimpleMonitorForm = () => {
+  const {
+    control,
+    register,
+    handleSubmit,
+    formState: { errors, isValid, isSubmitted },
+  } = useFormWrapped({
+    mode: 'onBlur',
+    reValidateMode: 'onChange',
+    shouldFocusError: true,
+    defaultValues: { urls: '', locations: [] as ServiceLocations },
+  });
+
+  const { application } = useKibana().services;
+
+  const [monitorData, setMonitorData] = useState<SimpleFormData>({
+    urls: '',
+    locations: [],
+  });
+
+  const onSubmit = (data: SimpleFormData) => {
+    setMonitorData(data);
+  };
+
+  const { data, loading } = useFetcher(() => {
+    if (!monitorData.urls || monitorData.locations.length < 1) {
```

Comment:
Seems odd to do validation here and on the main form. Can this level of validation be accomplished on the form, then only call the api when `isValid` is true?